token/cli/src/main.rs

```diff
@@ -2613,6 +2621,15 @@ fn app<'a, 'b>(
                             Rate authority defaults to the mint authority."
                         ),
                 )
+                .arg(
+                    Arg::with_name("metadata_address")
+                        .long("metadata-address")
+                        .value_name("METADATA_ADDRESS")
```

Comment:
nit: no need to show `--metadata-address METADATA_ADDRESS` in the help, so this can be simplified
```suggestion
                        .value_name("ADDRESS")
```